src/formatDate.js

```diff
@@ -8,7 +8,57 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const fromValue = fromFormat[fromFormat.length - 1];
```

Comment:
The variable `fromValue` is used to split the date string, assuming it is the last element of `fromFormat`. This might not always be correct if the format doesn't end with a separator. Consider validating the format or handling cases where the separator is not at the end.